src/test/kotlin/uk/gov/dluhc/printapi/service/SftpServiceTest.kt

```diff
@@ -31,11 +32,14 @@ internal class SftpServiceTest {
     @Mock
     private lateinit var sftpOutboundTemplate: SftpRemoteFileTemplate
 
+    @Mock
+    private lateinit var objectMapper: ObjectMapper
+
     private lateinit var sftpService: SftpService
 
     @BeforeEach
     fun setUp() {
-        sftpService = SftpService(sftpInboundTemplate, sftpOutboundTemplate)
+        sftpService = SftpService(sftpInboundTemplate, sftpOutboundTemplate, objectMapper)
```

Comment:
I know you didnt add this test class and BeforeEach hook; but I wonder why we are not using `@InjectMocks` ? 🤔

Reply:
Awesome; thank you 👍